src/ccl_power.c

```diff
@@ -15,8 +15,8 @@
 #include "ccl_emu17_params.h"
 
 
-/*------ ROUTINE: ccl_cosmology_compute_power_class -----
-INPUT: ccl_cosmology * cosmo
+/*------ ROUTINE: ccl_free_class_structs -----
+INPUT: ccl_cosmology * cosmo, ...
```

Comment:
We should finish this doc string...

Reply:
I agree, it's been like this for ages. I'll fix it up.